src/validations/wearable.ts

```diff
@@ -5,6 +5,27 @@ import { ADR_45_TIMESTAMP, calculateDeploymentSize, validateInRow } from '.'
 import { OK, Validation, validationFailed } from '../types'
 
 const wearableSizeLimitInMB = 2
+
+/** Validate wearable representations are referencing valid content */
+export const wearableRepresentationContent: Validation = {
+  validate: async ({ deployment }) => {
+    const { entity } = deployment
+    const wearableMetadata = entity.metadata as Wearable
+    const representations = wearableMetadata?.data?.representations
+    if (!representations) return validationFailed('No wearable representations found')
+    if (!entity.content) return validationFailed('No content found')
```

Comment:
There's also another check needed, the representations contain a `mainFile` property that should be included in the `representation.contents`.
https://github.com/decentraland/builder/blob/9492e08439981c71d312daf620b3181fe253c293/src/modules/item/types.ts#L70

Reply:
I believe it is validated in the wearable schema: https://github.com/decentraland/common-schemas/blob/main/src/platform/wearables/representation.ts#L66

Reply:
If that's the case, it seems to be validating everything that we need!